pandas/core/series.py

```diff
@@ -2128,8 +2131,8 @@ def quantile(self, q=0.5, interpolation="linear"):
 
         See Also
         --------
-        core.window.Rolling.quantile
-        numpy.percentile
+        core.window.Rolling.quantile : Calculate the %(name)s quantile.
```

Comment:
Does this get substituted? Could you try [buliding the docs](https://pandas.pydata.org/docs/development/contributing.html#building-the-documentation) and posting screenshots?

Reply:
Marco, thanks for your feedback. No, it doesn't get substituted. How shall I handle it?
![screenshot](https://user-images.githubusercontent.com/7809331/75097903-be9e8c00-558e-11ea-9c5d-27b23237c74f.png)

Reply:
Shouldn't in this case the text be: `core.window.Rolling.quantile : Calculate the rolling quantile.`?

Reply:
I think `Calculate the rolling quantile.` is ok. Not sure what you wanted to get from `%(name)s`, but we only use variables when we reuse the same doctring for different objects, and some parts change (which I don't think it applies here).